package.json

```diff
@@ -114,13 +114,12 @@
 		"prisma": "^5.21.1",
 		"prisma-json-types-generator": "^3.1.1",
 		"tsc-watch": "^6.2.0",
-		"typescript": "^5.4.5",
+		"typescript": "~5.4.5",
 		"vitest": "^2.1.3"
 	},
 	"resolutions": {
-		"@types/node": "20.11.5",
 		"ansi-regex": "^5.0.1",
-		"discord-api-types": "^0.37.98",
+		"discord-api-types": "0.37.97",
```

Comment:
I'm pretty sure latest djs is on latest -types (and with ^), unless that's still not released cc: @vladfrangu

Reply:
I checked what's in yarn.lock... :\